ext/date/php_date_arginfo.h

```diff
@@ -257,6 +257,10 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_class_DateTime_createFromImmutable, 0, 0, 1)
 	ZEND_ARG_OBJ_INFO(0, object, DateTimeImmutable, 0)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO_EX(arginfo_class_DateTime_createFromInterface, 0, 0, 1)
+                ZEND_ARG_OBJ_INFO(0, object, DateTimeInterface, 0)
+ZEND_END_ARG_INFO()
```

Comment:
This is a generated file, you need to edit `ext/date/php_date.stub.php` instead.

Reply:
@nikic thanks

Reply:
@nikic How do you generate that file after updating the stub one ?

Reply:
Should happen automatically when you run `make`, but if it doesn't you can use `php scripts/dev/gen_stub.php ext/date/php_date.stub.php`.

Reply:
Why is this generated file committed? ;-)

Reply:
The automatic generation relies on gmake pattern rules, which are not generally supported (e.g. not by nmake).

Reply:
The files require PHP to be generated, and we can't require PHP to build PHP ;) Same reason why zend_vm_execute.h is committed.